test.js

```diff
@@ -1,5 +1,11 @@
 var t = "hello";
 
-if (t === "hello') {
-    console.log('ok');
+if (t === "hello") {
+    var OK = 'OKAYY';
 }
+
+function showIt() {
+    return OK;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["OK" used outside of binding context.](https://www.codacy.com/app/rubstar/rvs-pr-testing/file/2496543359/issues/source?bid=3260975&fileBranchId=3269370#l8)